workflow/envs/bcftools.yaml

```diff
@@ -0,0 +1,6 @@
+channels:
+  - conda-forge
+  - bioconda
+  - nodefaults
+dependencies:
+  - bcftools =1.20
```

Comment:
Can we update the other stuff in tools such that no extra env is needed here?

Reply:
Done.